### PR TITLE
new features: postburst_slow_ahp_values, time_to_postburst_slow_ahp

### PR DESCRIPTION
**Pull request by AurelienJaquier**

### CODE REVIEW

**Comment by DrTaDa** [comment]
@AurelienJaquier could you remove this print ? https://github.com/BlueBrain/eFEL/blob/master/efel/cppcore/LibV1.cpp#L1964

**Comment by AurelienJaquier** [comment]
@DrTaDa done :)

**Comment by DrTaDa** [comment]
> @DrTaDa done :)

Thanks ! You'r the best

**Comment by AurelienJaquier** [comment]
This should solve Issue #282

File: efel/cppcore/LibV5.cpp
Comment (anilbey) [inline efel/cppcore/LibV5.cpp:4368]: Actually if this function just takes one more parameter that is the percentage (0.3 here, 0.25 in the other call) then it can be reused in 6 placed.

File: efel/cppcore/LibV5.cpp
Comment (anilbey) [inline efel/cppcore/LibV5.cpp:3667]: I think `std::bind2nd` is deprecated in C++11 and removed in C++17.
https://en.cppreference.com/w/cpp/utility/functional/bind12

There are lambda expressions available now for the replacement.
https://en.cppreference.com/w/cpp/language/lambda

```
t_start_index = find_if(
    t.begin() + peak_indices[burst_end_indices[i]],
    t.begin() + end_index,
    [t_start](double value) { return value >= t_start; }
) - t.begin();
```

**Review by anilbey — CHANGES_REQUESTED** [review]

File: efel/cppcore/LibV5.cpp
Comment (AurelienJaquier) [reply efel/cppcore/LibV5.cpp]: thanks, solved in last commit

**Comment by anilbey** [comment]
Thanks looks good to me. There is also the https://github.com/BlueBrain/eFEL/pull/334 PR which reduces the repetitiveness in the C++ code base.

For example, instead of having different functions for `inv_first_ISI`, `inv_second_ISI` ... `inv_fifth_ISI`, it just uses the following to have a single implementation.
```cpp
  FptrTableV5["inv_first_ISI"] = [](mapStr2intVec& intData,
                                    mapStr2doubleVec& doubleData,
                                    mapStr2Str& strData) {
    return LibV5::inv_ISI_generic(intData, doubleData, strData, 0);
  };
  FptrTableV5["inv_second_ISI"] = [](mapStr2intVec& intData,
                                     mapStr2doubleVec& doubleData,
                                     mapStr2Str& strData) {
    return LibV5::inv_ISI_generic(intData, doubleData, strData, 1);
  };
  FptrTableV5["inv_third_ISI"] = [](mapStr2intVec& intData,
                                    mapStr2doubleVec& doubleData,
                                    mapStr2Str& strData) {
    return LibV5::inv_ISI_generic(intData, doubleData, strData, 2);
  };
```

After #334 is merged, this PR can be written much shorter (perhaps even reduced by half). Would it be possible to hold off on merging this PR until after #334 is completed?

**Comment by AurelienJaquier** [comment]
@anilbey sure we can wait, no problem. This is waiting from April anyway

**Comment by AurelienJaquier** [comment]
> After #334 is merged, this PR can be written much shorter (perhaps even reduced by half).

Alright, done in latest commits.

This PR is ready to be merged on my side.

File: docs/source/eFeatures.rst
Comment (anilbey) [inline docs/source/eFeatures.rst:873]: Can these documentations be grouped similar to  ` Python efeature_ : inv_first_ISI, inv_second_ISI, inv_third_ISI, inv_fourth_ISI, inv_fifth_ISI, inv_last_ISI` to avoid copy pastes in the docs and the pseudocode? The pseudo code can have a variable for the percentage constant  perhaps?

Comment (AurelienJaquier) [reply docs/source/eFeatures.rst]: Good catch. Done in latest commit

File: tests/test_basic.py
Comment (AurelienJaquier) [reply tests/test_basic.py]: god catch, thanks! fixed in latest commit

File: docs/source/eFeatures.rst
Comment (AurelienJaquier) [reply docs/source/eFeatures.rst]: done in latest commit

**Review by anilbey — APPROVED** [review]
Thanks looks good to me

**Pull request merged**
